### PR TITLE
docs: add email list subscribe guide

### PR DESCRIPTION
**Pull request by iLern**
issue #257

Add mail subscribe way in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,25 @@ The functions of this system include but are not limited to:
 - [HugeGraph-Hubble](https://hugegraph.apache.org/docs/quickstart/hugegraph-hubble): HugeGraph-Hubble is HugeGraph's web visualization management platform, a one-stop visual analysis platform. The platform covers the whole process from data modeling, to rapid data import, to online and offline analysis of data, and unified management of graphs;
 - [HugeGraph-Tools](https://hugegraph.apache.org/docs/quickstart/hugegraph-tools): HugeGraph-Tools is HugeGraph's deployment and management tools, including functions such as managing graphs, backup/restore, Gremlin execution, etc.
 
+### Subscribe the mailing list
+Read all the mail easily at: https://lists.apache.org/list.html?dev@hugegraph.apache.org
+
+HugeGraph offers 2 dev mailing lists:
+- hugegraph-dev: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org) for development and users discussions.
+- hugegraph-commits: [commits@hugegraph.apache.org](mailto:commits@hugegraph.apache.org) for commits / [pull requests](https://github.com/apache/incubator-hugegraph/pulls) and other notifications like code review comments.
```
Comment (imbajin) [inline README.md:59]: normal user don't need to subscribe commits mailing list, could remove it

File: README.md [hunk 2]
```diff
@@ -51,6 +51,25 @@ The functions of this system include but are not limited to:
...
+You can subscribe to the mailing list anytime you want, and you can also check [historical emails](https://lists.apache.org/list.html?dev@hugegraph.apache.org).
+
+Subscribe to the mailing list by following steps:
+- Email [dev-subscribe@hugegraph.apache.org](mailto:dev-subscribe@hugegraph.apache.org) by your email account, and then you will receive a confirmation email.
+- Reply the confirmation email to confirm your subscription. Then, you will receive another email.
+- Now you are a subscriber of the mailing list. If you have more questions, just email to the mailing list and someone will reply you soon.
+- If you want to unsubscribe the mailing list, just email to [dev-unsubscribe@hugegraph.apache.org](mailto:dev-unsubscribe@hugegraph.apache.org) and follow the steps in the confirmation email.
+
+More information of mailing subscribe can be found at:
+- https://dubbo.apache.org/zh/docs/contribution-guidelines/contributor/mailing-list-subscription-guide_dev/
```
Comment (imbajin) [inline README.md:70]: remove dubbo refer (just leave it in issue)

File: README.md [hunk 3]
```diff
@@ -51,6 +51,25 @@ The functions of this system include but are not limited to:
...
 - [HugeGraph-Tools](https://hugegraph.apache.org/docs/quickstart/hugegraph-tools): HugeGraph-Tools is HugeGraph's deployment and management tools, including functions such as managing graphs, backup/restore, Gremlin execution, etc.
 
+### Subscribe the mailing list
+Read all the mail easily at: https://lists.apache.org/list.html?dev@hugegraph.apache.org
+
+HugeGraph offers 2 dev mailing lists:
+- hugegraph-dev: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org) for development and users discussions.
+- hugegraph-commits: [commits@hugegraph.apache.org](mailto:commits@hugegraph.apache.org) for commits / [pull requests](https://github.com/apache/incubator-hugegraph/pulls) and other notifications like code review comments.
+
+You can subscribe to the mailing list anytime you want, and you can also check [historical emails](https://lists.apache.org/list.html?dev@hugegraph.apache.org).
```
Comment (imbajin) [inline README.md:61]: good

enhance it to''check xx to view all the mails?''  (or use full)

File: README.md [hunk 4]
```diff
@@ -51,6 +51,25 @@ The functions of this system include but are not limited to:
...
+
+Subscribe to the mailing list by following steps:
+- Email [dev-subscribe@hugegraph.apache.org](mailto:dev-subscribe@hugegraph.apache.org) by your email account, and then you will receive a confirmation email.
```
Comment (imbajin) [inline README.md:64]: add the case if not receive the reply mail (refer the issue menthioned)

**Review by imbajin — COMMENTED** [review]
thanks, some comments

**Comment by liuxiaocs7** [comment]
Hi, @iLern, you could refer [this](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to link pr with associated issue, once the pr is merged the issue will be closed automatically.

**Comment by liuxiaocs7** [comment]
> I committed to my branch issue#257 but saw 1 workflow awaiting approval. Does it normal? Or did I do anything wrong?

For First-time contributors, CI will not be triggered automatically, it needs the manager of the repository to help trigger, just wait.

**Comment by liuxiaocs7** [comment]
Hi, @imbajin, should we update "Community" page at the same time?

**Comment by imbajin** [comment]
Some translation could be improved by `GPT/New Bing`, try to paste the phrase and ask it for the improvement~ 🤖 
image

<details>
<summary>GPT Improvement Example </summary>

Here are some suggestions for improving the markdown English description:

- In the first sentence, you can use "an email list" instead of "email list" to be more grammatically correct.
- In the second bullet point, you can use "for both development and user discussions" instead of "for development and users discussions" to avoid confusion between singular and plural forms.
- In the third sentence, you can use "the following steps" instead of "following steps" to be more specific.
- In the fourth sentence, you can use "a reply" instead of "the confirmation email" to avoid repetition.
- In the fifth sentence, you can use "another confirmation email" instead of "another email" to be more clear.
- In the seventh sentence, you can use "anytime you want" instead of "anytime you want" to be more natural.
- In the ninth bullet point, you can use "send an email" instead of "send mail" to be more consistent with the rest of the text.
- In the last sentence, you can use "More information about subscribing to mailing lists can be found at:" instead of "More information of mailing subscribe can be found at:" to be more grammatical and accurate.

The improved paragraph is:

### Subscribe to the mailing list

HugeGraph offers an email list for development and user discussions. 
- hugegraph-dev: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org) for both development and user discussions.

Subscribe to the mailing list by following these steps:
- Email [dev-subscribe@hugegraph.apache.org](mailto:dev-subscribe@hugegraph.apache.org) from your email account, and then you will receive a confirmation email.
- Reply to the confirmation email to confirm your subscription. Then, you will receive another confirmation email.
- Now you are a subscriber of the mailing list. If you have more questions, just email to the mailing list and someone will reply to you soon.
- If you want to unsubscribe from the mailing list, just email [dev-unsubscribe@hugegraph.apache.org](mailto:dev-unsubscribe@hugegraph.apache.org) and follow the steps in the confirmation email.

You can subscribe to the mailing list anytime you want. Additionally, you can check [historical emails / all emails](https://lists.apache.org/list.html?dev@hugegraph.apache.org) easily (even if you are not subscribed to the list).

Some notes:
- If you don't receive the confirmation email, please send an email after 24 hours later.
- Don't send an email to **dev** until you subscribe to the mailing list successfully (otherwise the email will be banned).

More information about subscribing to mailing lists can be found at:
- http://apache.org/foundation/mailinglists.html#subscribing

</details>


BTW, could click `Resolve Conversation` after u addressed the comment.

**Review by imbajin — APPROVED** [review]

**Comment by liuxiaocs7** [comment]
Nice Prompt, for easier reference, copy it here.

```
检查上面的markdown英文描述，是否有语病或可以改进的地方，请指出具体的点，最后输出改进后的完整段落（注意保留原格式）
```

**Pull request merged**